### DISCUSSION TITLE
IMPORT FAILED

### DISCUSSION THREAD

**Discussion by speeg** [crystian/ComfyUI-Crystools#94] [Q&A] [answered]
I have an issue.
![pyrsmi](https://github.com/user-attachments/assets/91387c5e-a740-4b5a-97e0-973a768bc9a2)
I also reinstalled but still have the same issue.
Please let me know how to fix it.

**Comment by crystian** [comment, marked as answer]
please, @speeg  try this:
on the folder of the extension: `ComfyUI\custom_nodes\ComfyUI-Crystools\`
run: `pip install -r requirements.txt`

please, tell me if it works, thanks!

**Reply by speeg** [reply]
Thank you for getting back to me.
I already did it but still.
However, it was solved through updates(1.16).
Thank you!

**Comment by SnowTung999** [comment]
i am also import failed. 
i try to install 
py -m pip install -r requirements.txt is not work
